client/my-sites/stats/features/modules/stats-locations/stats-locations.tsx

```diff
@@ -336,6 +368,7 @@ const StatsLocations: React.FC< StatsModuleLocationsProps > = ( {
 						heroElement={ heroElement }
 						mainItemLabel={ optionLabels[ selectedOption ]?.headerLabel }
 						toggleControl={ toggleControlComponent }
+						downloadCsv={ ! shouldGateTab ? downloadCsvElement : null }
```

Comment:
Thanks for the review, @kangzj! I just added this last commit because, during a final test before merging, I noticed that the button at the top was visible on the Regions and Cities tabs when gated. This simple fix ensures it stays hidden.

I will proceed with the merge once CI checks completes :)